pkg/specs/volumes.go

```diff
@@ -78,6 +81,67 @@ func createPostgresVolumes(cluster apiv1.Cluster, podName string) []corev1.Volum
 	return result
 }
 
+func createVolumesAndVolumeMountsForPostInitApplicationSQLRefs(
+	refs *apiv1.PostInitApplicationSQLRefs,
+) ([]corev1.Volume, []corev1.VolumeMount) {
+	length := len(refs.ConfigMapRefs) + len(refs.SecretRefs)
+	digitsCount := utils.CountDigits(length)
```

Comment:
here, we should use the inline code to get the count and will be easy to know how you get that number